tests/RustTest.sh

```diff
@@ -41,3 +41,7 @@ else
 fi
 
 cargo bench $TARGET_FLAG
+
+cd ..
+
+cargo test --manifest-path "../rust/flatbuffers/Cargo.toml"
```

Comment:
Is it possible to include this in the existing test code? AFAICT it fits into the file structure we already have.

Reply:
I moved build test into existing `tests/rust_usage_test`.